### ISSUE TITLE
[EXMO] fetch_balance() not complete

### ISSUE THREAD

**Issue by WoK-E** [ccxt/ccxt#5259]
- OS: Linux
- Programming Language version: PHP 7.2
- CCXT version: 1.18.643
- Exchange: EXMO
- Method: fetch_balance()

fetch_balance() is not filling the returned array completely. PTI is not included in the 'free' and 'used' arrays, and there is no 'PTI' array.

I have not checked that all other coins are included correctly.

**Comment by WoK-E** [comment]
PTI needs to be added to feesAndLimits

**Comment by kroitor** [comment]
The problem with EXMO is that their API is incomplete and does not serve those fees and limits nor does it serve the full market info from any of the _"official"_ API endpoints. We've tried scraping the webpage https://exmo.me/ctrl/feesAndLimits that contain the necessary info, however, the EXMO website bans users who are accessing it too often. Therefore we have to hardcode those fees and limits. And because the exchange introduces new pairs all the time, the hardcoded list gets outdated quickly. You can, however, force-load them from web (and avoid hardcoded pairs) by adding:

```PHP
$exchange = \ccxt\exmo (array(
    'options' => array (
        'useWebapiForFetchingFees' => true,
    ),
));
```

I've added the missing pairs to the hardcoded list and it will be available in 1.18.644 in 10 minutes.

Thx for reporting, appreciate your help!
